app/src/js/components/Granules/list.js

```diff
@@ -74,6 +77,16 @@ const AllGranules = ({
     setWorkflow(workflowOptions[0]);
   }, [workflowOptions]);
 
+  useEffect(() => {
+    dispatch(
+      getCount({
+        type: 'granules',
+        field: 'error.Error.keyword',
+        sidebarCount: false
+      })
+    );
+  }, [dispatch]);
```

Comment:
Originally I was thinking we'd need a new API to get the list of granule errors. I think we can just use the existing /stats/aggregates endpoint though. I'm bucketing the aggregates by the error type, then just getting a list of all of those buckets (ignoring the actual count).